### PR TITLE
refactor: Cleanup usage of core

### PR DESCRIPTION
**Pull request by atomrc**
Prepare for https://github.com/wireapp/wire-web-packages/pull/4650

### CODE REVIEW

File: src/script/client/ClientEntity.ts [hunk 1]
```diff
@@ -75,17 +75,6 @@ export class ClientEntity {
...
-  /**
-   * Splits an ID into user ID, client ID & domain (if any).
-   */
-  static dismantleUserClientId(id: string): {clientId: string; domain?: string; userId: string} {
-    // see https://regex101.com/r/c8FtCw/1
-    const regex = /((?<domain>.+)@)?(?<userId>.+)@(?<clientId>.+)$/g;
-    const match = regex.exec(id);
-    const {domain, userId, clientId} = match?.groups || {};
-    return {clientId, domain, userId};
-  }
```
Comment (atomrc) [inline src/script/client/ClientEntity.ts:87]: This has been moved to a util module

File: src/__mocks__/@wireapp/core.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+export class Account {
```
Comment (przemvs) [inline src/__mocks__/@wireapp/core.ts:20]: Maybe use name for mocks /Class/Mock or Mocked/Class/? WDYT?

Comment (atomrc) [reply src/__mocks__/@wireapp/core.ts:20]: It needs to be the same name as the class exported in the real package, actually.

Comment (przemvs) [reply src/__mocks__/@wireapp/core.ts:20]: Or it should have same name as original class ?

**Review by przemvs — APPROVED** [review]

**Pull request merged**